pkg/user/queri_params.go

```diff
@@ -0,0 +1,5 @@
+package user
+
+const (
+	Email = "email"
+)
```

Comment:
Please rename this file to query_params :)

Reply:
Sure, Done!